extensions/ql-vscode/src/extension.ts

```diff
@@ -83,7 +83,7 @@ import {
   ProgressReporter,
   queryServerLogger,
 } from "./common";
-import { QueryHistoryManager } from "./query-history";
+import { QueryHistoryManager } from "./query-history/query-history";
```

Comment:
This might be a good candidate for creating an `index.ts` file in the `query-history` directory so we can just import from `./query-history` instead of `./query-history/query-history`, but that's something better done in a follow-up PR if we want to do this.

Reply:
👍